pages/team.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: page
+title: MDAnalysis team
+---
+
+MDAnalysis is a community-driven project that is made possible through the efforts of many members who contribute in numerous and diverse ways, ranging from direct package development, maintenance, documentation, communication, and managerial responsibilities. On this page we list identified project roles, current leads and supporting team members for each of those roles, and those historically responsible for the tasks. We note that the listed roles on this page can differ significantly in scope and required effort.
+
+## Roles
+
+In this section we list major contributors who are leading or contributing substantial effort to the roles listed below. The **Current** column lists all those who are currently responsible for a particular role, from 2023–2024. In addition, we list past leading figures from 2020 onwards in the "Historical contributors" column. As this team organisation structure was started in November 2023, many people contributed to these tasks before then and may not appear in this table, in particular the [emeritus core developers]({{ site.url }}/about/#mdanalysis-emeriti-core-developers).
```

Comment:
```suggestion
In this section we list major contributors who are leading or contributing substantial effort to the roles listed below.

### The **Current** column

This lists all those who are currently responsible for a particular role, from November 14th 2023 to November 14th 2024.

### The "Historical 2021-2023" column

Here we list past leading figures from 2021 to November 14th 2023 in the "Historical lead contributors" column.

### A note on historical contributions

As this team organisation structure was started in November 2023, many people contributed to these tasks before then and may not appear in this table, in particular the [emeritus core developers]({{ site.url }}/about/#mdanalysis-emeriti-core-developers).
```

Reply:
I like breaking down the definitions of the columns. Can we link each definition to the table's column heading via a hyperlinked footnote?